plugins/scaffolder/package.json

```diff
@@ -56,9 +56,9 @@
     "@material-ui/lab": "4.0.0-alpha.57",
     "@react-hookz/web": "^20.0.0",
     "@rjsf/core": "^3.2.1",
-    "@rjsf/core-v5": "npm:@rjsf/core@5.0.0-beta.16",
```

Comment:
I actually think that these deps can be removed now as I think `FormProps` is exported from the `scaffolder-react` package entirely.